vite.main.config.ts

```
@@ -28,11 +28,77 @@
+          try {
+            const jsonPath = pkgJsonPath(name);
+            const json = JSON.parse(fs.readFileSync(jsonPath, "utf8"));
+            copyPackage(name);
+            const deps = Object.assign({}, json.dependencies || {}, json.optionalDependencies || {});
+            for (const depName of Object.keys(deps)) {
+              // Skip builtins and electron
+              if (/^(electron|node:|fs|path|crypto|stream|events|url|buffer)$/.test(depName)) continue;
+              walk(depName);
+            }
+          } catch (e) {
+            console.warn(`⚠️  Failed to process ${name}: ${String((e as any)?.message || e)}`);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)